code/modules/mining/machine_redemption.dm

```diff
@@ -160,7 +160,7 @@
 		return
 	message_sent = TRUE
 	var/area/A = get_area(src)
-	var/msg = "Now available in [A]:<br>"
+	var/msg = "Now available in [A]:\n"
```

Comment:
This doesnt really fix the problem, so there isnt much point of adding the `\n`. Just remove it all.

Reply:
Alright. Still learning etc but seems like simply deleting vs putting \n are the same thing. 

https://i.imgur.com/dR0FHNR.png (seems similar)

I have deleted them and made another comit.